src/navigation/router-config.tsx

```diff
@@ -4,9 +4,11 @@ import { IRouterConfig } from './types';
 
 const DirectionView = lazy(() => import('../modules/direction/components/DirectionView'));
 const MainView = lazy(() => import('../modules/main/components/MainView'));
+const Header = lazy(() => import('../modules/main/components/Header/Header'));
```

Comment:
header should be present on the page nomatter the route, therefore, it should not be in the touter